composer.json

```diff
@@ -24,10 +24,5 @@
     },
     "autoload-dev": {
         "classmap": ["test/src"]
-    },
-    "config": {
-        "platform": {
-            "php": "5.3.99"
```

Comment:
I'd prefer to update this instead of deleting it. The new value should be `7.1.99`.

Having it here ensures that even if I run `composer update` under PHP 7.3, the lock file will only ever store dependencies that are at least compatible with PHP 7.1, which is important for running the 7.1 tests.

Reply:
Libraries shouldn't have a lock file to begin with. I didn't remove the lock file because I'm trying to keep this PR as focused as possible, but if you insist, I will put this back in.

Reply:
I'd appreciate it. I'm aware of the arguments against committing lock files, but it's not something I want to change in this PR, if at all.